lint_test.go

```diff
@@ -7,5 +7,5 @@ import (
 )
 
 func TestAll(t *testing.T) {
-	testutil.TestAll(t, Funcs)
+	testutil.TestAll(t, Funcs, "")
```

Comment:
`testutil.TestAll` is [not documented](https://github.com/dominikh/go-lint/commit/a730e73f0085f274d67b586a1d587fd6e42e6708#commitcomment-18386854), so it's not possible to know whether the empty string is a valid/supported value.

It happens to work with current code, and is equivalent to "." directory because `filepath.Clean` [ignores all empty strings](https://godoc.org/path/filepath#Join). So `filepath.Join("testdata", "")` is the same as `filepath.Join("testdata", ".")`, both return just `"testdata"`.